Library/Homebrew/os/linux.rb

```diff
@@ -25,6 +27,23 @@ def os_version
         "Unknown"
       end
     end
+
+    sig { returns(T::Boolean) }
+    def wsl?
+      /-microsoft/i.match?(OS.kernel_version.to_s)
+    end
+
+    sig { returns(Version) }
+    def wsl_version
+      Version::NULL unless wsl?
```

Comment:
```suggestion
      return Version::NULL unless wsl?
```

or this line is a no-op